tools/XCConfigExporter/Sources/CompilationDatabaseToolkit/KWBackgroundTask.m

```diff
@@ -0,0 +1,106 @@
+//
+// Licensed under the terms in License.txt
+//
+// Copyright 2010 Allen Ding. All rights reserved.
+//
+// This file is licensed under a BSD equivalent license.
+
+#import "KWBackgroundTask.h"
+
+NSString *const NSTaskDidTerminateNotification;
+
+static NSString *const KWTaskDidTerminateNotification = @"KWTaskDidTerminateNotification";
+
+static NSString *const KWBackgroundTaskException = @"KWBackgroundTaskException";
+
+@implementation KWBackgroundTask
```

Comment:
This should be replaced by the task runner we use in XCHammer if we want to keep it around

Reply:
Do we actually need this dependency?